sonar-plugin-api/src/main/java/org/sonar/api/batch/postjob/issue/PostJobIssue.java

```diff
@@ -62,24 +60,18 @@
   Integer line();
 
   /**
-   * Effort to fix the issue. Used by technical debt model.
-   */
-  @CheckForNull
-  Double effortToFix();
-
-  /**
    * Message of the issue.
    */
   @CheckForNull
   String message();
 
   /**
-   * Severity.
+   * Severity.  Only accurate in preview/issues mode.
    */
   Severity severity();
 
   /**
-   * If the issue a new one.
+   * If the issue a new one. Only available in preview/issues mode.
```

Comment:
what is the behavior if issues mode is disabled ? I suggest to throw IllegalStateException.